### PR TITLE
Add Flag for Host to Enabled/Disable Bidder from Usersync

### PR DESCRIPTION
**Pull request by AlexBVolcy**
Addresses this issue: https://github.com/prebid/prebid-server/issues/2573

### CODE REVIEW

File: usersync/chooser_test.go [hunk 1]
```diff
@@ -67,11 +69,16 @@ func TestChooserChoose(t *testing.T) {
 
 	cooperativeConfig := Cooperative{Enabled: true}
 
+	var usersyncDisabled *bool
+	enabled := false
+	usersyncDisabled = &enabled
```
Comment (VeronikaSolovei9) [inline usersync/chooser_test.go:74]: Can be replaced with `ptrutil.ToPtr(false)`

File: config/bidderinfo.go [hunk 2]
```diff
@@ -124,6 +124,9 @@ type Syncer struct {
 
 	// SupportCORS identifies if CORS is supported for the user syncing endpoints.
 	SupportCORS *bool `yaml:"supportCors" mapstructure:"support_cors"`
+
+	// TODO: Add description
+	Enabled *bool `yaml:"enabled" mapstructure:"enabled"`
```
Comment (VeronikaSolovei9) [inline config/bidderinfo.go:129]: Please add description

File: endpoints/cookie_sync.go [hunk 3]
```diff
@@ -490,6 +490,8 @@ func getDebugMessage(status usersync.Status) string {
 		return "No sync config"
 	case usersync.StatusTypeNotSupported:
 		return "Type not supported"
+	case usersync.StatusBlockedByDisabledUsersync:
+		return "Status blocked by disabled usersync" // TODO: Check Java
```
Comment (VeronikaSolovei9) [inline endpoints/cookie_sync.go:494]: Is this TODO needed?

**Review by VeronikaSolovei9 — COMMENTED** [review]
Added minor comments, local testing looks good

**Review by guscarreon — APPROVED** [review]
LGTM

**Pull request merged**